src/sentry/snuba/metrics/datasource.py

```diff
@@ -379,7 +385,7 @@ def get_series(projects: Sequence[Project], query: QueryDefinition) -> dict:
         orderby_field = [key for key, value in query.fields.items() if value == query.orderby[0]][0]
         query.fields = {orderby_field: parse_field(orderby_field)}
 
-        snuba_queries = SnubaQueryBuilder(projects, query).get_snuba_queries()
+        snuba_queries, _ = SnubaQueryBuilder(projects, query).get_snuba_queries()
```

Comment:
Let me look into it

Reply:
tl;dr we chatted about this, and it probably doesn't make a different because there is a subsequent try and catch so it will either hit the catch block at https://github.com/getsentry/sentry/pull/32829/files#diff-fe042572eb7930f0cd4d0ce5b501b92fe1fd200e29a24b86004245745d8e8812R410 and in that case `results` will be empty, and so it won't matter anyway or it will be set at https://github.com/getsentry/sentry/pull/32829/files#diff-fe042572eb7930f0cd4d0ce5b501b92fe1fd200e29a24b86004245745d8e8812R423